ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -3126,6 +3130,7 @@ static int zend_jit_set_ip(dasm_State **Dst, const zend_op *opline)
 
 static int zend_jit_set_ip_ex(dasm_State **Dst, const zend_op *opline, bool set_ip_reg)
 {
+	|	ENDBR
```

Comment:
``ENDBR`` here is definitely wrong. Please identify, the code that causes the problem here.